rshim.spec.in

```diff
@@ -143,9 +145,13 @@ fi
 %{_mandir}/man8/bfb-install.8.gz
 %{_mandir}/man8/bfb-tool.8.gz
 %{_mandir}/man8/bf-reg.8.gz
+%{_mandir}/man8/bf-pldm-ver.8.gz
 %{_datadir}/selinux/packages/rshim-fix.te
 
 %changelog
+* Thu Jul 03 2025 Shih-Yi Chen <shihyic@nvidia.com> - 2.4.3
+- Add bf-pldm-ver script to get all package version information from PLDM image
+
```

Comment:
no need to change the changelog section. We only change it when building a version, not for every commit.